html_parser.js

```diff
@@ -184,14 +184,19 @@ function get_termid(season, year) {
     // As different technology will likely be used in 81 years, this is a valid assumption
     let termid = "20" + year;
     switch(season) {
-        case "FL":
-            return termid + "40";
-        case "SP":
+	case "FL": 
+	    // Fall term: associated year is the same year as the following 
+	    // Spring term as per SearchNEU conventions
+	    termid = "20" + (year + 1);
             return termid + "10";
-        case "S1":
-            return termid + "20";
-        case "S2":
+        case "SP": // Spring term
             return termid + "30";
+        case "S1": // Summer 1 term
+            return termid + "40";
+        case "S2": // Summer 2 term
+            return termid + "60";
+	case "SM": // Full Summer term (typically reserved for graduate courses)
```

Comment:
... but here you say "SM" = summer full. Since these are values used by the HTML audit itself, they should be decided by looking at the HTML. 

If we need someone who has taken summer full courses, I know Max Daniels is taking summer full courses.

Reply:
update: I just emailed Max. I'll slack you when he responds.